install.sh

```diff
@@ -218,7 +214,14 @@ function extract_install () {
     #extract and install
     echo_intra "Extracting ${1} ..."
     if [[ "$2" == *".zst" ]];then
-      LD_LIBRARY_PATH=${CREW_PREFIX}/lib${LIB_SUFFIX}:/lib${LIB_SUFFIX} tar -Izstd -xpf ../"${2}"
+      if [[ -e /usr/bin/zstd ]]; then
+        PATH=/usr/bin:$PATH tar -Izstd -xpf ../"${2}"
```

Comment:
Will `tar -I/usr/bin/zstd -xpf ../"${2}"` better?

Reply:
We shouldn't be setting `LD_LIBRARY_PATH` in the first place.

Reply:
It's a very narrow situation where that is happening.